lib/local-links-manager/check_links/link_status_updater.rb

```diff
@@ -3,29 +3,27 @@
 module LocalLinksManager
   module CheckLinks
     class LinkStatusUpdater
```

Comment:
This looks good.  We should probably change similar code in the HomepageStatusUpdater class too so that it's consistent.